src/aura/STG_CMP_Tabs/STG_CMP_Tabs.cmp

```diff
@@ -38,6 +38,23 @@
     <aura:handler event="c:STG_EVT_Save" action="{!c.saveSettings}" />
     <aura:handler event="c:STG_EVT_Cancel" action="{!c.resetSettings}" />
 
+    <div class="slds-hide" id="successToast" aura:id="successToast">
+        <div class="slds-box">
+            <div class="slds-notify_container slds-notify_container--inline">
+                <div class="slds-notify slds-notify--toast slds-theme--success" role="alert">
+                    <span class="slds-assistive-text">Success</span>
+                    <button class="slds-button slds-button--icon-inverse slds-notify__close" tabindex="-1">
+                        <c:svgIcon name="close" category="utility" size="small" assistiveText="Close" pressIcon="{!c.clostToast}" />
+                    </button>
+                    <div class="slds-notify__content slds-grid">
+                        <div class="slds-col slds-align-middle">
+                            <h2 class="slds-text-heading_small">Settings Saved Successfully.</h2>
```

Comment:
@lifewithryan I think this should be a new custom label.

Reply:
Ooh, and maybe a review from @juliesnow here?

Reply:
Oh yea, good catch!!